pom.xml

```
@@ -29,7 +29,8 @@
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
     <spotbugs.skip>true</spotbugs.skip>
-    <jenkins.version>2.475</jenkins.version>
+    <!-- TODO https://github.com/jenkinsci/jenkins/pull/9696 -->
+    <jenkins.version>2.476-rc35330.1b_8c1984dc3b_</jenkins.version>
```

Release is available

```suggestion
    <jenkins.version>2.476</jenkins.version>
```